addon/components/ember-table.js

```diff
@@ -128,6 +129,18 @@ export default class EmberTable2 extends Component {
     return Number.isInteger(numFixedColumns) && numFixedColumns !== 0;
   }
 
+  /**
+   * Estimated height for each row.
+   */
+  @computed('staticHeight')
```

Comment:
We should match the API in vertical-collection. `estimateRowHeight` is a number representing the height. `staticHeight` is a boolean changing the mode of the table.

Reply:
I don't think we should have 2 variables in this case. One should be enough. User does not need to know about vertical collection implementation details.